doc/setup_procedure.md

```diff
@@ -0,0 +1,84 @@
+# Setup Procedure
+
+Person 1 and person 2 work in parallel
+
+## Person 1 (Ciaran or Orion)
+
+1. Open the Deck box on table in front of the pilot.
+
+1. Look through the Lexan sheet on the deck box and check for any connectors that have fallen out.
+
+    * [wiring diagram](example.com)
```

Comment:
Does this need to get updated now, or later? I'm thinking if its later we might well forget about it.

Reply:
the SID isnt good enough so I need to make a wiring diagram of the deck box for there

Reply:
i'd like to leave it to remind me to do it